include/deal.II/matrix_free/operators.h

```diff
@@ -1512,7 +1606,7 @@ namespace MatrixFreeOperators
                     const std::pair<unsigned int,unsigned int>  &cell_range) const
   {
     typedef typename Base<dim,VectorType>::value_type Number;
-    FEEvaluation<dim,fe_degree,n_q_points_1d,n_components,Number> phi (data);
+    FEEvaluation<dim,fe_degree,n_q_points_1d,n_components,Number> phi (data, this->selected_rows[0]);
     for (unsigned int cell=cell_range.first; cell<cell_range.second; ++cell)
```

Comment:
I guess we it would be a good idea to document that `MassOperator`and `LaplaceOperator` always only represent an operator for the first selected block.